openedx/features/learner_profile/templates/learner_profile/learner-achievements-fragment.html

```diff
@@ -11,7 +11,7 @@
 
 <div class="learner-achievements">
     % if course_certificates or own_profile:
-        <h3 class="u-field-title">Course Certificates</h3>
+        <h3 class="u-field-title">${_("Course Certificates")}</h3>
```

Comment:
This new term hasn't appeared in po files, it probably should

Reply:
It is there though. Translation already existed, I just used gettext to get it